packages/at_secondary_server/lib/src/verb/handler/otp_verb_handler.dart

```diff
@@ -57,10 +55,12 @@ class OtpVerbHandler extends AbstractVerbHandler {
           throw InvalidRequestException(
               'Client not allowed to not store semi permanent pass code');
         }
-        String? otp = verbParams['otp'];
-        await keyStore.put(
-            'private:spp.$otpNamespace${AtSecondaryServerImpl.getInstance().currentAtSign}',
-            AtData()..data = otp);
+        int sppTtl = int.tryParse(verbParams[AtConstants.ttl] ?? '') ?? -1;
```

Comment:
This is the second of the two actual changes in this PR - use ttl from verbParams if supplied, else explicitly set to -1